### PR TITLE
MAISTRA-723 map gateway ports 80/443 to 8080/8443 to remove need for …

### PR DESCRIPTION
**Pull request by rcernich**
…anyuid SCC

Signed-off-by: rcernich <rcernich@redhat.com>

This needs to be merged after maistra/istio#40.

### CODE REVIEW

File: tmp/build/patch-charts.sh [hunk 1]
```diff
@@ -248,9 +261,7 @@ function patchTemplates() {
 \1\2/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
 
   # change the location of the healthCheckFile from /health to /tmp/health
-  if [[ "${COMMUNITY,,}" != "true" ]]; then
-    sed -i -e 's/\/health/\/tmp\/health/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
-  fi
+  sed -i -e 's/\/health/\/tmp\/health/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
```
Comment (jwendell) [inline tmp/build/patch-charts.sh:262]: is this change supposed to go in this PR?

**Review by jwendell — APPROVED** [review]

File: tmp/build/patch-charts.sh (hunk 1 above)
Comment (rcernich) [reply tmp/build/patch-charts.sh:262]: Yes.  Community images don't work without it for some reason.  Health checks were failing on sidecar injector pod.

Comment (dmitri-d) [reply tmp/build/patch-charts.sh:262]: yep:

```
Readiness probe failed: Error: fail on inspecting path /health: stat /health: no such file or directory 
```

**Review by dmitri-d — APPROVED** [review]
lgtm

File: tmp/build/patch-charts.sh (hunk 1 above)
Comment (jwendell) [reply tmp/build/patch-charts.sh:262]: I was not asking whether this is valid or not. I was asking if this should belong to this PR or another. I'm happy with it, but it seems it's a different problem than this PR fixes.

**Review by brian-avery — APPROVED** [review]

File: tmp/build/patch-charts.sh (hunk 1 above)
Comment (rcernich) [reply tmp/build/patch-charts.sh:262]: I don't know how you want to look at it, but I couldn't test the changes without this.  Picnic area/campground rule: leave it cleaner than you found it.

**Pull request merged**